prow/cmd/deck/static/script.js

```diff
@@ -499,8 +502,8 @@ function redraw(fz) {
         builds.appendChild(r);
     }
     const jobCount = document.getElementById("job-count");
-    jobCount.textContent = "Shows " + totalJob + " job(s)";
-    drawJobBar(totalJob, jobCountMap);
+    jobCount.textContent = "Shows " + Math.min(totalJob, 500) + "/" + totalJob + " jobs";
+    drawJobBar(Math.min(totalJob, 500), jobCountMap);
```

Comment:
I think we still just want to use `totalJob` here. The count at the top can show the overall info now that it is clear that only some jobs are displayed. Overall stats will be more useful and accurate than stats for just the truncated list of jobs.